webserver.py

```diff
@@ -121,6 +121,8 @@ def get_admin_data():
             cursor.execute('SELECT password FROM admin;')
             result = cursor.fetchone()
             data['password'] = result[0]
+
+            data['recipes'] = RECIPE_CONFIG
```

Comment:
If you want to avoid the long variable references in your template, you can pass the RECIPE_CONFIG in separately from the admin data.

So your render_template would look something like
```
render_template('admin.html', data=get_admin_data(), recipes=RECIPE_CONFIG['recipes'])
```

then in the template you could just use `recipes` instead of `data['recipes'].recipes`